sdk/go/common/workspace/paths.go

```diff
@@ -164,6 +164,26 @@ func DetectPolicyPackPathFrom(path string) (string, error) {
 	})
 }
 
+// DetectPolicyPackPathAt locates the PulumiPolicy file at the given path. If no project is found, an empty path is
+// returned. Unlike DetectPolicyPackPathFrom, this function does not search upwards in the directory hierarchy.
+func DetectPolicyPackPathAt(path string) (string, error) {
+	for _, ext := range encoding.Exts {
+		packPath := filepath.Join(path, PolicyPackFile+ext)
+		info, err := os.Stat(packPath)
+		if err != nil {
+			if errors.Is(err, fs.ErrNotExist) {
+				continue
+			}
+			return "", err
+		}
+		if info.IsDir() {
+			return "", fmt.Errorf("found directory %s, but expected a file", packPath)
```

Comment:
Is this really an error? Surely we just return empty here that a policy pack wasn't found?

Reply:
Yeah that makes more sense. It's a bit weird if it's a directory but not an error.